### DISCUSSION TITLE
1st Grand Gathering - 인과추론팀 부스 방명록

### DISCUSSION THREAD

**Discussion by jsshin2022** [CausalInferenceLab/Causal-Inference-Study-for-Analyst#50] [Show and tell]
[신진수/ 크래프톤] 부스 방명록 예시글 입니다.

**Comment by ehddnr301** [comment]
[이동욱/ 위메이드] 부스 너무 유익해요🥰

**Reply by cjkangme** [reply]
[강창진] 인과추론이 정확히 무엇인지 잘 몰랐는데 잘 설명해주셨습니다.

**Comment by wjdghwo** [comment]
[정호재/ 롯데캐피탈] 인과추론팀 화이팅!!!

**Comment by soheekim911** [comment]
[김소희/티빙] 인과추론 ♥ 실무 응원합니다!

**Comment by soonhp** [comment]
인과추론 궁금했었는데 응원합니다!

**Comment by leobumjin** [comment]
인과추론분들 너무 친절하고 전통이 깊은 팀 답게 부스분들 너무 멋있어요!!🤭😀😀

**Comment by yjhyo88** [comment]
진수님의 추진력은 항상 대단한거 같습니다 ㅎㅎ 인과추론팀 화이팅!

**Comment by yunhee1** [comment]
인과추론에 대해 자세히 알 수 있어 좋았습니다 :)

**Comment by Generoustandard** [comment]
정호재님의 열정적인 인과추론 설명에 대해서 감명 깊게 들었습니다. 실무에 필요한 철학적인 내용을 잘 설명을 해주신 것 같아요. 데이터 분석에 필수적인 부분이라고 생각합니다.

**Comment by brian604** [comment]
[김지원 / 포트래이] 데이터 기반으로 결정을 할 수 있다는 부분, 비즈니스에 많이 필요하.. 아니 모두에게 필요할것 같습니다!
나중에는 인과추론 모델이 모두에게 배포되는 그날까지 ...

**Comment by DavidPark96** [comment]
인과추론에 대해 어떻게 스터디가 진행되는지 궁금했는데, 친절하게 설명해주셔서 감사합니다! 기회가 된다면 꼭 참여하고싶네요 :)

**Comment by baekdusan** [comment]
인과추론에 대한 다양한 인사이트를 얻을 수 있었습니다! 감사합니다.

**Comment by hyeji-K** [comment]
인과추론 너무 궁금했었는데 설명해주셔서 좋았습니다!

**Comment by Kim-Minhee** [comment]
잘 몰랐던 분야였는데 친절한 설명 감사합니당~

**Comment by imyeeun** [comment]
[리멤버/임예은]
인과추론 10기에는 참여하고 싶어요!!

**Comment by Ronal92** [comment]
인과추론 공부만 해오다가 현장에서 어떻게 쓰이는지 너무 궁금했는데 바쁘신데도 자세히 얘기해주셔서 감사합니다!!!
잠깐이었지만 많은 도움이 되었어요 ㅎㅎㅎㅎ
디스코드에 여러 사례들 올려주시면 꼭 참고하겠습니다👍

**Comment by Homes201** [comment]
[하상집] 인과추론 스터디에 대해 알 수 있어서 좋았습니다.

**Comment by yunha-p** [comment]
사용자경험 전공자로서 흥미가 가는 부스였습니다! 참여할 수 있는 기회가 있다면 추후에 참여해보고 싶어요 ㅎㅎ 👍

**Comment by kudokiring97** [comment]
인과추론팀에 관심이 생기게 되었습니다.
청강후, 내년 스터디 참여 해볼까 합니다!

**Comment by HyeonseokSim** [comment]
[심현석/학생] 통계학을 전공으로 공부하면서 상관관계과 인과관계는 서로 다른 개념이란 것을 귀 아플 정도로 들었고, 주변에서 인과추론을 많이 공부하고 있단 소식을 많이 들어서 저도 관심을 갖게 되었습니다. 머신러닝, 딥러닝에서 인과추론의 역할이 무엇인지? A/B 테스트와도 연관이 있는지? 이런 부분들을 질문드렸는데 친절히 설명해주셔서 너무 좋았습니다!!! 노션에 공부하신 자료들도 업로드 되어있다 해주셔서 많이 공부하겠습니다:)

**Comment by LeoSeminLee96** [comment]
[이세민/ 가짜연구소] 인과추론 수도콘 강의와 및 서적을 늘 keep up하고 있습니다!

**Comment by geonhak904** [comment]
인과추론에 대한 설명 감사합니다

**Comment by statsops** [comment]
안녕하세요 인과추론 왔어요!

**Comment by Kdreamtomaster** [comment]
[강신우 / 구직자] 인과추론 입문계기에 대해서 실무적 경험과 연관지어 설명해주시어서 좋았습니다 감사합니다

**Comment by Hitdahit2** [comment]
[박주영/코어라인소프트] 부스가 재밌습니다!

**Comment by Kim-Ju-won** [comment]
인과추론에 대한 자세한 설명 및 자료를 제공해주셔서 감사합니다~!

**Comment by nbril0313** [comment]
인과추론에 대해 관심 가질 수 있었던 계기를
가지게 되어 좋았습니다

**Comment by qjrm1430** [comment]
[박중헌/중헌아카이브,취준생] 인과추론에 대해 잘 설명해주셔거 유익한 시간이됐습니다!

**Comment by ehyun0910** [comment]
인과추론 부스 방명록 남깁니다 !!

**Comment by HyojinKim88** [comment]
인과추론팀 반갑습니다!

**Comment by MetaDatamong** [comment]
인과추론에 대해 잘배우고 갑니다~

**Comment by JeongSuJin** [comment]
(정수진)감사합니다!

**Comment by j1s000** [comment]
유익한 내용 감사합니다!

**Comment by DrawingProcess** [comment]
감사합니다! 파이팅입니다!!

**Comment by leh60245** [comment]
[이의형/한양대ERICA캠퍼스] 인과추론을 처음 알아갑니다..!

**Comment by Acolyte-Noway** [comment]
[조준형/ 9기 수치해석 러너] 인과추론에 관심이 있었는데 너무 좋은 계기가 되었습니다‼️‼️

데이터 분석가 희망하는데 필요한 인사이트라 생각이 됩니다. 좋은 스터디 열어주셔서 감사합니다😊

**Comment by clastro** [comment]
궁금한 내용 답변 감사합니다!

**Comment by beomhwan9052** [comment]
[함범환/무직] 인과추론이라는 분야에 대해서 알게되서 너무 좋았고, 한번 배워보고 싶다는 생각이 들었습니다!

**Comment by hojae-m-choi** [comment]
[최호재/샤페론] 인과추론 너무 공부하고 싶어요 😭

**Comment by euisuk-chung** [comment]
[정의석/LG전자] 책 이미 구매해서 잘 보고 있습니다🫡🫡 화이팅입니다!! 스터디도 하고 싶네요 ㅠㅠㅠ

**Comment by HongYuBinBin** [comment]
인과추론 관심있는데 설명 감사합니다ㅎㅎ

**Comment by Hyundd** [comment]
인과추론 더 자세히 알고싶어요!!

**Comment by thsamajiki** [comment]
인과추론 많이 배우고 싶어요!

**Comment by raylee0519** [comment]
[이정우/동국대] 인과추론에 대한 개념을 많이 들을 수 있었습니다. 인과추론이 우리 일상생활에 많이 녹아들어있었네요 🙂

**Comment by Ko-Jiho** [comment]
[고지호 / 러너] 항상 인과추론 곁눈질로 보면서 궁금했었는데, 직접 이야기 들으면서 정보 얻을 수 있어서 너무 좋았습니다! 기회되면 추후에 인과추론 연구소도 들어갈 수 있으면 좋겠습니다 :)
